.github/workflows/merge.yaml

```diff
@@ -0,0 +1,185 @@
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Fetch all the tags
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8.10 # Patch version must be specified to avoid any cache confusion, since the cache key depends on the full Python version. If left unspecified, different patch versions could be allocated between jobs, and any such difference would lead to a cache not found error.
+
+      - id: stop-early
+        run: if "${GITHUB_WORKSPACE}/.github/has-functional-changes.sh" ; then echo "::set-output name=status::success" ; fi # The `check-for-functional-changes` job should always succeed regardless of the `has-functional-changes` script's exit code. Consequently, we do not use that exit code to trigger deploy, but rather a dedicated output variable `status`, to avoid a job failure if the exit code is different from 0. Conversely, if the job fails the entire workflow would be marked as `failed` which is disturbing for contributors.
+
+  publish-to-pypi:
+    runs-on: ubuntu-20.04
+    needs: [ check-for-functional-changes ]
+    if: needs.check-for-functional-changes.outputs.status == 'success'
+    env:
+      PYPI_USERNAME: openfisca-bot
+      PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      CIRCLE_TOKEN: ${{ secrets.CIRCLECI_V1_OPENFISCADOC_TOKEN }} # Personal API token created in CircleCI to grant full read and write permissions
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Fetch all the tags
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8.10 # Patch version must be specified to avoid any cache confusion, since the cache key depends on the full Python version. If left unspecified, different patch versions could be allocated between jobs, and any such difference would lead to a cache not found error.
+
+      - name: Cache deps
+        uses: actions/cache@v3
+        with:
+          path: venv
+          key: deps-ubuntu-20.04-np1.20.3-py3.8.10-${{ hashFiles('setup.py') }}
+
+      - name: Cache build
+        uses: actions/cache@v3
+        with:
+          path: venv/**/[oO]pen[fF]isca*
+          key: build-ubuntu-20.04-np1.20.3-py3.8.10-${{ hashFiles('setup.py') }}-${{ github.sha }}
+
+      - name: Cache release
+        uses: actions/cache@v3
+        with:
+          path: dist
+          key: release-ubuntu-20.04-np1.20.3-py3.8.10-${{ hashFiles('setup.py') }}-${{ github.sha }}
+
+      - name: Upload package to PyPi
+        run: |
+          source venv/bin/activate
+          make publish
+
+      - name: Update doc
+        run: |
+          curl -X POST --header "Content-Type: application/json" -d '{"branch":"master"}' https://circleci.com/api/v1.1/project/github/openfisca/openfisca-doc/build?circle-token=${{ secrets.CIRCLECI_V1_OPENFISCADOC_TOKEN }}
+
+  publish-to-conda:
```

Comment:
After the merge of this PR, I will update my PR specific to Conda : https://github.com/openfisca/openfisca-core/pull/1173